### PR TITLE
Create candidate-sebgoa.md

### PR DESCRIPTION
**Pull request by sebgoa**
I think I will need to ask for an exception as the Grafana dashboard only shows 48 contributions in the last year.

### CODE REVIEW

File: elections/2021-SC/candidate-sebgoa.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+-------------------------------------------------------------
+name: Sebastien Goasguen
+ID: sebgoa
+info:
+  - affiliation: TriggerMesh
+-------------------------------------------------------------
+
+## Contributions to Knative (list):
+
+* Involved with project since 2018 and the inital announcement at Google Next.
```
Comment (dprotaso) [inline elections/2021-SC/candidate-sebgoa.md:10]:
```suggestion
* Involved with project since 2018 and the initial announcement at Google Next.
```

**Comment by geekygirldawn** [comment]
> I think I will need to ask for an exception as the Grafana dashboard only shows 48 contributions in the last year.

Yes, please file an exception in Elekto if you haven’t already.

**Comment by sebgoa** [comment]
Actually according to the latest dashboard I should be eligible now with 51 contributions: 

https://knative.teststats.cncf.io/d/9/developer-activity-counts-by-repository-group-table?orgId=1&var-period_name=Last%20year

**Comment by odacremolbap** [comment]
🔝  :heart: :yin_yang:  such a joy having Sebastien candidacy here!
He is a long distance runner in the open source world, very strategic thinker and great listener and communicator.

**Comment by csantanapr** [comment]
I didn't see until now. 

+1 Seb would be a great addition to SC

**Comment by cmoulliard** [comment]
Sebastien is a great open source evangelist that I know personally since a couple of years and Apache contributor since long time ago. He has all the skills needed to contribute to the SC :-)

 +1

**Comment by matzew** [comment]
+1 on this nomination. Also all the good work that Seb and team (TriggerMesh) did in the past already for Knative, and being directly involved

**Pull request merged**